assets/scss/donate.scss

```diff
@@ -0,0 +1,20 @@
+@import "_functions";
+@import "_mixins";
+@import "_variables";
+
+.block-lb-donate.block {
```

Comment:
https://git.drupalcode.org/project/y_donate/-/blob/2.0.x/modules/lb_donate/assets/scss/lb-donate.scss?ref_type=heads

Reply:
@anpolimus 
https://git.drupalcode.org/project/y_donate/-/merge_requests/14

Reply:
https://www.drupal.org/project/y_donate/issues/3438104 merged. Same as above, please remember to bump the library version.